.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - 2.6
+  - 2.7
+addons:
+  apt:
+    packages:
+      - tcl8.5
+install:
+    # install easybuild-framework (and dependencies)
+    # use framework 'develop' branch, except when testing 'master'
+    - if [ "x$TRAVIS_BRANCH" = 'xmaster' ]; then FRAMEWORK_BRANCH=master; else FRAMEWORK_BRANCH=develop; fi
+    - easy_install https://github.com/hpcugent/easybuild-framework/archive/${FRAMEWORK_BRANCH}.tar.gz
+    # install environment modules tool using 'install_eb_dep.sh' script provided by easybuild-framework
+    - source $(which install_eb_dep.sh) modules-3.2.10 $HOME && source $MOD_INIT
```

Comment:
For the easyblock unit tests, it doesn't matter at all, no module commands are run.

I'm even wondering now whether this is needed at all...
Maybe it needs to be there to make the framework happy.